test/infrastructure/inmemory/pkg/server/proxy/dial.go

```diff
@@ -33,85 +33,80 @@ import (
 
 const defaultTimeout = 10 * time.Second
 
-// Dialer creates connections using Kubernetes API Server port-forwarding.
+// Dialer opens connections via Kubernetes API server port-forward.
```

Comment:
Fair to fix this, I just want to note that we ever only use it in unit tests for the inmemory provider afaik.